### ISSUE TITLE
Provide better field, field descriptions for UIForm  

### ISSUE THREAD

**Issue by LiteSun** [apache/apisix-dashboard#1724]
# Feature request

## Please describe your feature
Provide better field name, field descriptions for UIForm.

- [ ] https://github.com/apache/apisix-dashboard/pull/1718#discussion_r610282445
- [ ] https://github.com/apache/apisix-dashboard/pull/1725#discussion_r611281834
- [ ] https://github.com/apache/apisix-dashboard/pull/1725#discussion_r611281834

**Comment by starsz** [comment]
Oh. Can you add more description about your feature?
The PR is merged. Can we close this issue?

**Comment by LiteSun** [comment]
No, there is a series of plug-in forms that need to be changed.

**Comment by juzhiyuan** [comment]
labels needed
